docs/RELEASE_ISSUE_TEMPLATE.md

```diff
@@ -70,7 +71,7 @@ Checklist:
 - [ ] **Stage 1 - Internal Testing**
   - [ ] CHANGELOG.md has been updated
     - use [`./bin/mkreleaselog`](https://github.com/ipfs/go-ipfs/tree/master/bin/mkreleaselog) to generate a nice starter list
```

Comment:
`mkreleaselog` is not working. It expects the source code to be inside $GO_HOME/src, and also needs zsh to work properly. We can change this to use the `Generate release notes` button here: https://github.com/ipfs/kubo/releases/edit/v0.15.0-rc1